contracts/adapters/AlpacaVaultAdapter.sol

```diff
@@ -48,6 +48,20 @@ contract AlpacaVaultAdapter is IVaultAdapterV2 {
   /// @dev The router path to sell alpaca for BUSD
   address[] public path;
 
+  /// @dev lastUpdated
+  uint256 public lastUpdated;
+
+  modifier checkoutWithdraw() {
```

Comment:
This modifier also has to be added on the function totalValue()

Reply:
If we added this to totalValue, this function had to be non-view because of state change. And we have to update vaultv2 because there is another totalValue().